src/styles/functions/_max-width.scss

```diff
@@ -4,8 +4,10 @@
   $value-type: type_of($value);
 
   @if $value-type != number {
-    @warn "Expected $value #{$value} [#{$value-type}] to be of type Number.";
+    // @warn "Expected $value #{$value} [#{$value-type}] to be of type Number.";
```

Comment:
please don't remove `@warn`, these are intentional to avoid wrong applications of our functions and help to keep the quality high

Reply:
It should not warn me in quiet mode. do you know why?

Reply:
Alright.
So please never "fix" symptoms, that smells. But rather fix the real cause, which are two here IMHO:

- some existing sass code is buggy and causes these warnings
- regarding `--quite` it's a tooling issue (if it's broken at all). Also keep in mind that the canonical implementation of sass is moving away from ruby to [dart, and the API is changing!](https://github.com/sass/dart-sass)

Regarding verbosity standards, there isn't just one log level, an many tools offer a granular increase of verbosity, like `-v`, `-vv`, `-vvv`...
Also verbosity does not mean we do not log anything.

Reply:
We both tried and we saw that fixing the root needs more investigation in this case and as we can see it has not been solved so far. 

Living with symptoms is worse than hiding them, they are costing us a lot of time and money. So "fixing/hiding" them temporarily still helps more than just living with them.

Reply:
> We both tried

Nope, I have not tried it, You tried and failed

> Living with symptoms is worse than hiding them, they are costing us a lot of time and money

A log costs you time and money, really? Not for me

Reply:
@AndyOGo and @jackblackCH 

I personally think that lots of logs is like not having any log cause those relevant to you are not visible.

Now, I looked in a 15 min timebox over the quite flag for sass and "failed" as well (15 min timebox). Even `./node_modules/.bin/node-sass src/app/index.scss -o dist/ -q` will not suppress warnings so there must be something not working on node-sass side

@AndyOGo remeber we have a similar talk about `verbose` as well on the frontend-lib side? Where  for me the tons of output made it not usable anymore.

The golden 2 solutions would be:

- Resolve the `--quite` issue
- Resolve the origin of the warning.

Now i suggest that, if you have time, you look into it. As me and @jackblackCH couldnt fix it in a reasonable time due to prio issue i think makes the most sense that you fix it. Plus, you know best as well on why the warn output is there since a year.

I dont know which coding approach is "better": lots of logs for lots of info or only those that are severe. But as we code in a Team, lets decide democratically on which approach we go as we have to feel confortable with it.

I vote for leaving it commented out, create a "high-prio" issue and you fix it when @AndyOGo has time as this warning is here since more than a year. Would that be ok?

But thank you both for the very intense Idea sharing. Togheter we are better and from each other we learn.

Reply:
> A log costs you time and money, really? Not for me.
This is very rude and selfish and just harms the team. Development is not a one-man show,.

What if the majority is slower? ...

Reply:
As said I'm only concerned about removing warning/error logs.
Totally okay to put informational logs behind a `--verbose` flag, we can also discuss verbosity levels.

But this concrete `@warn` should never happen, and hence I favour either fix correctly here or in another PR.

Reply:
> This is very rude and selfish and just harms the team:

Alright, but exaggerated, incorrect or opinionated statements are constructive?

> What if the majority is slower by having to go through log they did not produce?

Trust me, `95%` of your time you will not deal with the build logs.
Again, I totally agree to remove all verbose informational logs, but not important `@warn`ings.

> Development is not a one-man show

Obviously it is not, who said it is?

Reply:
Ok @AndyOGo and @jackblackCH Please lets be constructive. Both ways are right and wrong andy. It a mattrer on how we behave in the team. So no-one cares about those wanrs cause no one looks at them and they are only visible to patterns lib dev (unless someone includes the SCSS). Please lets discuss this in a meeting. All 3 of us have lots of experience and opinion about stuff. We need to find a common ground which does not mean Andy, Lucas or Marcs way. I will setup a meeting. 

For the moment, lets merge this PR as the benefit it brings is more then the negative. I Personally create an issue with a DUE date and either me or @AndyOGo will fix the reason of the warning. If per that due date is not fixed, we will comment it back in.

Reply:
The majority has spoken, that's find with me.
Though I do not recommend this approach.

Sure all the rest is a great improvement I'm looking forward.